src/main/scala/sss/db/Db.scala

```diff
@@ -68,14 +66,17 @@ class Db(dbConfig: DbConfig)(closeableDataSource:CloseableDataSource, ec: Execut
 
   TL;DR for blockchain type applications views are a good solution.
    */
-  def view(name: String): View = new View(name, syncRunContext, dbConfig.freeBlobsEarly)
+  def view(name: String): View = viewWhere(name, where())
```

Comment:
Sorry I didn't spot this yesterday, but I think we can use a default param rather than selectWhere and viewWhere. 
`where` is a reasonable param to `select`. Slightly suprising as a param to `view` to it makes sense on inspection.